### DISCUSSION TITLE
Can I import a ".bash_profile" from my terminal use?

### DISCUSSION THREAD

**Discussion by boettchs** [Eugeny/tabby#8387] [Q&A] [answered]
I have both a bash and zsh profile that had aliases...can I import those into Tabby?
And BTW, excellent app!  Many thanks!

Scott

**Comment by Eugeny** [comment, marked as answer]
Hi Scott, Tabby doesn't have its own bash/zsh configuration and will use whatever you have in your home directly (it's just running `bash`/`zsh` inside the terminal)

**Reply by boettchs** [reply]
Thank you.  I'm just not sure how to use those profiles I have in there...within Tabby.  Could you guide me?
Thank you for the reply!!!

Scott

**Reply by boettchs** [reply]
Thanks, I figured it out.  Forgot I put the default to zsh and I wanted bash.  Changed it and all is good!
Cheers!
